nimbus-core/src/main/java/com/antheminc/oss/nimbus/domain/model/config/ModelConfig.java

```diff
@@ -31,6 +31,10 @@
 	public String getDomainLifecycle();
 	
 	public Repo getRepo();
+	
+	public void setRemote(boolean remote);
```

Comment:
Can the "setRemote" method not be exposed out via the interface? Would recommend to restrict methods that allow consumers of this API to change content within config.. config once created is treated static.